src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: ($day-size * $row-numbers) + 10px;
+  padding: $paddings;
+  gap: 1px;
+
+  &__day {
+    display: flex;
+    // box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+
+    @for $date from 1 through $calendar-days {
+      &:nth-child(#{$date})::before {
+        content: '#{$date}';
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color:pink;
+      transform: translateY(-20px);
```

Comment:
your animation ends immediately when i leave cursor from block, it must be smooth

Reply:
No problem, it makes sens... but the task doesn't say anything about it.